ci/container/Dockerfile

```diff
@@ -166,21 +168,30 @@ ENV PYTHONPATH=/ic/ci/src:/ic/ci/src/dependencies:$PYTHONPATH
 RUN cd /tmp/bazel && bazel version
 
 # Add Rust/Cargo support
-ARG RUST_VERSION=1.82.0
+RUN mkdir -p /tmp/rust-version/
```

Comment:
We should consolidate all these `RUN` layers into one layer.

Reply:
This particular one I'm not sure it's possible, since I think we need the directory to exist before the `COPY`. Happy to squash the later ones if you feel strongly about this.

Reply:
I don't care too much. Do you prefer if the file is written directly in `/tmp`?

Reply:
I prefer if we have less layers. It doesn't make any difference with space but logical grouping is better visually. I'm fine if it stays as it is as well. We should eventually spend time and energy with making the overall image smaller.